### DISCUSSION TITLE
How to display on 2 lines in the Window Title: Menu and Map filename?

### DISCUSSION THREAD

**Discussion by ldmpub** [freeplane/freeplane#834] [Help] [answered]
Hi all,

Yet the "Window title of Freeplane" contains on the same first line both menus and map filename ... see below ...

![image](https://user-images.githubusercontent.com/1452146/202853707-ece08f7a-ca83-4de2-898e-3dc64d098976.png)

**Is there a way to have:**
**- line 1: FP menus**
**- line 2: Map Path and filename.**

Thanks for your answers,

Note: despite browsing and searching in preferences, I didn't find how to perform this appearance customization, hoping it's not a feature request 😋.

**Comment by dpolivaev** [comment, marked as answer]
Does it depend on the chosen look and feel?

**Comment by ldmpub** [comment]
@dpolivaev You are right => display in 1 or 2 lines depends on "Theme"
![image](https://user-images.githubusercontent.com/1452146/202855038-9f2a03c6-b89c-413a-9f85-1363c8aac608.png)

=> solved, many thanks 👏.

**Reply by dpolivaev** [reply]
I think the flat look-and-feels put the lines together and they have a property to configure it but Freeplane does not set it
